x-pack/test/case_api_integration/common/lib/utils.ts

```diff
@@ -605,6 +609,74 @@ export const getConnectorMappingsFromES = async ({ es }: { es: KibanaClient }) =
   return mappings;
 };
 
+type ESConnectorFields = Array<{
```

Comment:
These make the integration test types easier. Since these types are no longer defined in the cases common area we need to define them here so we can access them.

Reply:
I think is ok to import them from `x-pack/plugins/cases/server/services/cases`

Reply:
Oh, I'll give that a shot, I think it was giving me an error, I suppose for this we could just ignore the eslint. Good point.